src/vcpkg/build.cpp

```diff
@@ -914,11 +913,11 @@ namespace vcpkg
             return_code = cmd_execute_and_stream_data(
                 command,
                 [&](StringView sv) {
-                    print2(sv);
-                    Checks::check_exit(VCPKG_LINE_INFO,
-                                       out_file.write(sv.data(), 1, sv.size()) == sv.size(),
-                                       "Error occurred while writing '%s'",
-                                       stdoutlog);
+                    msg::println(LocalizedString::from_raw(sv));
```

Comment:
```suggestion
                    msg::write_unlocalized_text_to_stdout(sv);
```

This is actually broken because you're adding additional newlines that aren't present in the chunk.